alpha/hal.cpp

```diff
@@ -324,7 +328,47 @@ void _TS_HAL::log_init()
 #endif
 }
 
-
+//
+// WIFI setup
+// 
+void _TS_HAL::wifi_init()
```

Comment:
Lets put all the wifi things into a file called radio.h/cpp.
Eventually BT things should move there as well.

wifi should be able to turn on/off on-demand, no  need to init in setup()